services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from db.models import MovieSession, Movie, CinemaHall
+from django.db.models import QuerySet
+import datetime
+
+
+def create_movie_session(
+    movie_show_time: datetime,

```

Comment:
The parameter `movie_show_time` is typed as `datetime`, which is a module, not a type. It should be `datetime.datetime` to correctly specify the type of the parameter.